### PR TITLE
* Replaced library and fix the MissingInformationToCreateTransaction bug in AfterPay

### PR DESCRIPTION
**Pull request by LeasoftKonstantinShapoval**
…aller.

### CODE REVIEW

**Review by tiutiunnykleasoft — APPROVED** [review]
OK.

**Comment by LeasoftKonstantinShapoval** [comment]
Complete PLUG-349

File: emspayafterpay/emspayafterpay.php [hunk 1]
```diff
@@ -376,7 +376,8 @@ public function getCustomerInformation($cart, $locale = '')
             'gender' => $gender,
             'birthdate' => $presta_customer->birthday,
             'ip_address' => Tools::getRemoteAddr(),
-            'locale' => $locale
+            'locale' => $locale,
+            'additional_addresses' => $this->getBillingAdress($cart)
```
Comment (tiutiunnykleasoft) [inline emspayafterpay/emspayafterpay.php]: getBillingAdress function not founded
the word Address must be two 'd' letters

**Review by tiutiunnykleasoft — CHANGES_REQUESTED** [review]
Not Ok but Ok

**Comment by LeasoftKonstantinShapoval** [comment]
Fixed

**Review by tiutiunnykleasoft — APPROVED** [review]

**Pull request merged**